doc/source/whatsnew/v0.17.0.txt

```diff
@@ -334,6 +335,62 @@ Google BigQuery Enhancements
 - The ``generate_bq_schema()`` function is now deprecated and will be removed in a future version (:issue:`11121`)
 - Update the gbq module to support Python 3 (:issue:`11094`).
 
+.. _whatsnew_0170.east_asian_width:
+
+Display Alignemnt with Unicode East Asian Width
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+
+   Enabling this option will affect to print DataFrame and Series performance (about 2 times slower).
```

Comment:
``Enabling this optino will affect the performance for printing of DataFrame and Series. ``